src/test/kotlin/racingcar/domain/CarsTest.kt

```diff
@@ -14,10 +14,23 @@ internal class CarsTest {
     @Test
     internal fun `가장 멀리 간 자동차가 우승자고 우승자는 한 명 이상일 수 있다`() {
         val cars = Cars(listOf("가", "나", "다"))
-        cars[0].move()
-        cars[1].move()
-        cars[2].stop()
-        assertThat(cars.winners(1))
+        cars[0].moveOrStop(shouldMove = true)
+        cars[1].moveOrStop(shouldMove = true)
+        cars[2].moveOrStop(shouldMove = false)
+        assertThat(cars.winners())
             .containsExactly(Car("가"), Car("나"))
     }
+
+    @Test
+    internal fun `moveOrStopAll 테스트`() {
+        val cars = Cars(listOf("가", "나", "다"))
+        assertThat(cars.apply { moveOrStopAll { false } })
+            .allSatisfy {
+                assertThat(it.position(1)).isEqualTo(0)
+            }
+        assertThat(cars.apply { moveOrStopAll { true } })
+            .allSatisfy {
+                assertThat(it.position(2)).isEqualTo(1)
+            }
```

Comment:
구현 코드뿐만 아니라 테스트 코드도 깔끔하게 유지하고 개선해야 합니다. 아래의 코드로 개선해 보면 어떨까요?

```suggestion
        cars.moveOrStopAll { false }
        assertThat(cars).allSatisfy {
            assertThat(it.position(1)).isEqualTo(0)
        }
        cars.moveOrStopAll { true }
        assertThat(cars).allSatisfy {
            assertThat(it.position(2)).isEqualTo(1)
        }
```